class.jetpack.php

```diff
@@ -1478,11 +1503,12 @@ public static function get_avatar_url( $id_or_email, $size = 96, $default = '',
 	 * array (
 	 *      'plugins'                       => (int) Number of plugin updates available.
 	 *      'themes'                        => (int) Number of theme updates available.
-	 *      'wordpress'                     => (int) Number of WordPress core updates available.
+	 *      'WordPress'                     => (int) Number of WordPress core updates available. // phpcs:ignore WordPress.WP.CapitalPDangit
```

Comment:
It looks like WordPress is still capitalized.

```suggestion
	 *      'wordpress'                     => (int) Number of WordPress core updates available. // phpcs:ignore WordPress.WP.CapitalPDangit.Misspelled
```